examples/ice-phases/README.md

```diff
@@ -5,8 +5,8 @@ Nature Communicationsvolume 9, Article number: 2173 (2018)
 #
 * step 1
 
-gen_soap_descriptors.py -fxyz ice-dataset.xyz --rcut 4 --n 4 --l 6 --g 0.2 --periodic True --output xyz
+python ../../scripts/gen_soap_descriptors.py -fxyz ice-dataset.xyz --rcut 4 --n 4 --l 6 --g 0.2 --periodic True --output xyz
 
 * step 2
```

Comment:
We decided to just use the name of the script, instead of the relative path in the examples,
because we expect people to install it